### DISCUSSION TITLE
ESP32   SHT30 verschwindet dirpareted

### DISCUSSION THREAD

**Discussion by darryl70** [arendst/Tasmota#15852] [Q&A] [answered]
![printscr](https://user-images.githubusercontent.com/83500719/175297485-a020221e-5f9c-44ac-95ed-a10fbff4c153.jpg)
Hallo  
Ich habe ein ESP32 4 Channel Relay Board (ESP32R4) mit Tasmota 12.02 und SHT30. Der SHT 30 wird erkannt verschwindet aber nach ein paar Sekunden. Ich kann die Anzeige nur durch  Neustart wider herstellen.  Anschluss header LCD 
https://robotdyn.com/catalog/smarthome/esp32-4-relays-controller-board-for-wi-fi-or-bluetooth-smart-home-system-equipment-control-programmable-esp32-4-relays-controller-board-for-wi-fi-or-bluetooth-smart-home-system-equipment-control-programmable-esp32-4-relays-controller-boa

Hi
I have an ESP32 4 Channel Relay Board (ESP32R4) running Tasmota 12.02 and SHT30. The SHT 30 is recognized but disappears after a few seconds. I can only restore the display by restarting. Connection header LCD

thank vor zour help danke 
![printscr](https://user-images.githubusercontent.com/83500719/175298152-839b010d-588d-477c-bbf3-2c0fb353e58f.jpg)

**Comment by Jason2866** [comment]
Which GPIOs are used? PullUp resistors for I2C bus installed? Cable length?

**Comment by darryl70** [comment, marked as answer]
SDA  GPIO 21
SCL GPIO 22
pull up on module selon description \
lengt 2 m

**Reply by darryl70** [reply]
pullup 9,9k

**Reply by Jason2866** [reply]
Issue found, I2C with a 2m cable!! I2C is a bus designed for a few cm.
